k8s-deployment-manifest-templates/deployment-mode/daemonset/container-insights-monitoring/fluent-bit/fluent-bit.yaml

```diff
@@ -231,6 +222,14 @@ data:
         Format              json
         Time_Key            time
         Time_Format         %Y-%m-%dT%H:%M:%S.%LZ
+    
+    [PARSER]
+        Name                cri
+        Format              regex
+        Regex               ^(?<time>[^ ]+) (?<stream>stdout|stderr) (?<logtag>[^ ]*) (?<message>.*)$
+        Time_Key            time
+        Time_Format         %Y-%m-%dT%H:%M:%S.%L%z
+        Time_Keep           On
```

Comment:
This part, I'm not entirely sure about but I took it from https://github.com/fluent/fluent-bit/blob/078226fbe1210110410da659d5b4af475cbd7279/conf/parsers.conf#L114-L121

Reply:
@SaxyPandaBear multiline parsers are different than normal parsers. The `multiline.parser    docker, cri` => this uses multiline parsers that are built into the FLB code. You don't need to define them.

Reply:
Ack, I wasn't sure if I needed to or not. I can remove

Reply:
Done